source/common/http/conn_manager_config.h

```diff
@@ -106,9 +106,9 @@ struct ConnectionManagerTracingStats {
 struct TracingConnectionManagerConfig {
   Tracing::OperationName operation_name_;
   std::vector<Http::LowerCaseString> request_headers_for_tags_;
-  uint64_t client_sampling_;
-  uint64_t random_sampling_;
-  uint64_t overall_sampling_;
+  envoy::type::FractionalPercent client_sampling_;
```

Comment:
Can we add a test here somewhere to make sure we get the default denominator correct in this conversion? It might be here but I couldn't find it quickly.

Reply:
Added more tests to check for defaults and configured values.